plugins/modules/net_tools/nmcli.py

```diff
@@ -709,6 +709,15 @@ class Nmcli(object):
     platform = 'Generic'
     distribution = None
 
+    wifi_sec_secret_options = [
+        '802-11-wireless-security.leap-password',
+        '802-11-wireless-security.psk',
+        '802-11-wireless-security.wep-key0',
+        '802-11-wireless-security.wep-key1',
+        '802-11-wireless-security.wep-key2',
+        '802-11-wireless-security.wep-key3'
+    ]
```

Comment:
```suggestion
    WIFI_SEC_SECRET_OPTIONS = (
        '802-11-wireless-security.leap-password',
        '802-11-wireless-security.psk',
        '802-11-wireless-security.wep-key0',
        '802-11-wireless-security.wep-key1',
        '802-11-wireless-security.wep-key2',
        '802-11-wireless-security.wep-key3'
    )
```
I think it's safer to have class constants be a little more obvious.

Reply:
Yeah, I took the hint from the casing of the other two constants. Whichever you prefer.
https://github.com/ansible-collections/community.general/blob/364f49f1f966d8f0a89e98020f4070f8004b8663/plugins/modules/net_tools/nmcli.py#L709-L710

Reply:
The value should also be a `tuple` instead of a `list`.